README.md

```diff
@@ -18,17 +18,23 @@ Thanks!
 
 ## Install
 ---------
-- ` git clone https://github.com/johnomernik/qgrid`
+In one command:
+
+`pip install git+https://github.com/JohnOmernik/qgrid  https://github.com/JohnOmernik/qgrid/raw/main/qgrid2-1.1.3-py3-none-any.whl`
+
+..or install from sources (in develop mode with `-e`):
+
+- `git clone https://github.com/johnomernik/qgrid --depth=1`
 - `cd qgrid`
-- `python setup.py install`
+- `pip install -e .`
 - `pip install qgrid2-1.1.3-py3-none-any.whl`
 - `cd ..`
 
 At this point, it worked for me with:
 
 - Python 3.9
 - JupyterLab 3.2.1
-- ipywidgets 7.6.5
+- ipywidgets 7.6.5 <8
```

Comment:
It worked for me on `jupyterlab==3.6.2` and `ipywidgets==7.7.4`, too. But `ipywidgets==8.0.5` broke it with a JavaScript error.

Reply:
You mean augment this PR with the latest versions of the other jupyter stuff that are compatible?

Reply:
I guess yes, it could be useful for new users to know the current latest compatible versions of other libraries so they don't spend the trouble to downgrade when not necessary.

Reply:
We don't really know which is the very latest version of each dependency before it breaks - we have to wait for the new releases to arrive. 

What i can do, and it is actually [suggested by the official docs]([Requirements](https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/#id6)), is to separate the `setup. Py:install_requires` from the `requirements, txt` file, and keep the capping only on the former while kissing all dependency versions in the later.